admin/settings/class-admin-apple-settings-section-advanced.php

```diff
@@ -60,7 +60,7 @@ public function __construct( $page ) {
 			'in_article_position' => [
 				'label'       => __( 'Position of In Article Module', 'apple-news' ),
 				'type'        => 'number',
-				'min'         => 0,
+				'min'         => 3,
```

Comment:
does any documentation need to be updated as a result of this change?

Reply:
Good question! This is part of a new, unreleased feature, and we will document it when we release v2.5.0 in May.